docs/upgrade/UPGRADE-v7.3.0.md

```diff
@@ -160,8 +152,8 @@ There you can find links to upgrade notes for other versions too.
         - `(test-)replace-domains-urls` was deprecated, use `(test-)domains-urls-replace` instead
         - `(test-)load-plugin-demo-data` was deprecated, use `(test-)plugin-demo-data-load` instead
         - don't forget to update your Dockerfiles, Kubernetes manifests, scripts and other files that might reference the phing targets above
-- we recommend upgrading PHPStan to level 4 [#1040](https://github.com/shopsys/shopsys/pull/1040)
```

Comment:
Link leads to "level 4" and that is confusing... Remove the link maybe?

Reply:
link removed